src/homework-01/basic_types.ts

```diff
@@ -1,116 +1,120 @@
 class School {
-  directions = [];
+  directions: Direction[] = [];
 
-  addDirection(direction) {
+  addDirection(direction: Direction): void {
     this.directions.push(direction);
   }
 }
 
 class Direction {
-  levels = [];
+  levels: Level[] = [];
+  _name: string;
 
-  get name() {
+  get name(): string {
     return this._name;
   }
 
-  constructor(name) {
+  constructor(name: string) {
     this._name = name;
   }
 
-  addLevel(level) {
+  addLevel(level: Level) {

```

Comment:
Please, do not forget to add explicitly type annotation for the function return type.